layouts/partials/blocks/testimonials.html

```diff
@@ -0,0 +1,77 @@
+{{ with .block }}
+  <section
+    class="min-h-[65vh] bg-black bg-opacity-10 py-20"
+    data-ga-category="links"
+  >
+    <div class="mx-auto max-w-6xl px-5">
+      <div class="lg:flex">
+        <h2
+          class="lg:sideways-heading mb-4 text-4xl font-black uppercase leading-none text-s-7 opacity-40 md:text-6xl lg:mb-0 lg:rotate-180"
+        >
+          {{ .title }}
+        </h2>
+        <div class="mt-8 flex" x-data="carousel()">
+          {{ with .entries }}
+
+            <div class="hidden h-64 w-12 md:flex">
+              <span @click="slide('left', 2)">
+                <svg
+                  class="h-12 w-12 self-center fill-s-2 opacity-75 drop-shadow filter lg:h-16 lg:w-16"
+                >
+                  <use xlink:href="#left-svg"></use>
+                </svg>
+              </span>
+            </div>
+            <ul
+              class="flex-no-wrap hide-scrollbars -mx-2 flex flex-1 overflow-y-hidden overflow-x-scroll md:-mx-4"
+              x-ref="carousel"
+            >
+              {{ range . }}
+                <li
+                  class="group mb-2 inline-block w-2/3 flex-none cursor-pointer snap-center px-2 pr-2 pb-2 focus:outline-none sm:pr-8 md:mr-12 md:w-1/2 md:px-4"
+                >
+                  <p>{{- .testimonial -}}</p>
+                </li>
+              {{ end }}
+
+            </ul>
+            <div class="hidden h-64 w-12 md:flex">
+              <span @click="slide('right', 2)">
+                <svg
+                  class="h-12 w-12 self-center fill-s-2 opacity-75 drop-shadow filter lg:h-16 lg:w-16"
+                >
+                  <use xlink:href="#right-svg"></use>
+                </svg>
+              </span>
+            </div>
+          {{ end }}
+
+        </div>
+
+        <script>
+          function carousel() {
```

Comment:
You can register this with Alpine so you don’t have to put the script in the footer here. See https://github.com/spotlightpa/poor-richard/blob/master/src/esbuild/enhancements.js .

Reply:
Added to enhancements

https://github.com/spotlightpa/poor-richard/pull/338/commits/f4fb2b50a4fc4e12c464adbba4363f56fbbe7eb9